system/ee/ExpressionEngine/Controller/Utilities/ImportConverter.php

```diff
@@ -142,6 +142,7 @@ public function index()
         );
 
         ee()->cp->render('settings/form', $vars);
+       
```

Comment:
Can you remove these extra lines - just want to make sure small edits don't add any unnecessary changes.